crypto/fipsmodule/sha/sha_test.cc

```diff
@@ -27,35 +27,93 @@ TEST(SHATest, SHA1ABI) {
   SHA1_Init(&ctx);
```

Comment:
line 24 `# if` was changed upstream. Without the change, it's not matching `#endif // SUPPORTS_ABI_TEST` on l. 119 and maybe not all tests are executed.

Reply:
For ppc64le, the [`SUPPORTS_ABI_TEST` macro is defined](https://github.com/aws/aws-lc/blob/31db68195bc1bb7542904b48ff8d770a8f920283/crypto/test/abi_test.h#L265), but its implementation of [`sha1_block_data_order` is in C here](https://github.com/aws/aws-lc/blob/0541314f0c8ab0e6e4955e70cb31747732797a9c/crypto/fipsmodule/sha/sha1-altivec.c#L188). 

Are the "SHAxxxABI" tests needed on PowerPC64LE?